src/renderers/art/ReactART.js

```diff
@@ -93,7 +93,7 @@ function injectAfter(parentNode, referenceNode, node) {
 
 // ContainerMixin for components that can hold ART nodes
 
-const ContainerMixin = assign({}, ReactMultiChild, {
+const ContainerMixin = assign({}, ReactMultiChild.prototype, {
 
```

Comment:
Will this affect releasing https://github.com/facebook/react/pull/7649 in a minor release?